lib/service_extensions.dart

```diff
@@ -107,3 +107,17 @@ const slowAnimations = ServiceExtensionDescription<num>(
 // This extension should never be displayed as a button so does not need a
 // ServiceExtensionDescription object.
 const String didSendFirstFrameEvent = 'ext.flutter.didSendFirstFrameEvent';
+
+// If you add a service extension to this file whose state should be restored on
+// page refresh/initial startup, add the extension to this list.
+const List<String> statefulExtensionsWhitelist = [
+  'ext.flutter.debugAllowBanner',
```

Comment:
done.

Reply:
fwiw, dart2js doesn't get much or any benefit out of objects being const so we shouldn't worry about making things const unless there is no cost to us to make them const or it is a case like flutter widget trees where cost is important to avoid creating a ton of dupe objects.